.travis.yml

```diff
@@ -0,0 +1,57 @@
+dist: focal
```

Comment:
We've moved to Github Actions from Travis - can you translate this to GH Actions?

Reply:
Alternatively this could be factored out into a later PR adding the YAML files for GH Actions